### DISCUSSION TITLE
Mobile app crashes on loading specific repo banner

### DISCUSSION THREAD

**Discussion by DasLixou** [community/community#78826] [Mobile] [answered]
https://github.com/community/community/assets/82600264/49fd00c4-4053-4349-99a4-391d884764eb

When I tracked it down right the problem seems to be when loading the banner image for ratatui-rs (could be wrong, just my assumption).

The app hangs and just crashes. Can reproduce it 100 times in a row, also after restart. Had the problem for a longer time

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.141.0 (116980603)
Device: iPhone 7 (iOS 15.8)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @DasLixou thanks for writing in. We are aware of this issue and are actively looking into it. We expect to have it fixed in the next few releases. Please bear with us and thanks again for the report!

**Reply by DasLixou** [reply]
@stevepopovich nope. Ratatui-rs was in Trends today again and it still crashes. Can't get over the 5th element (ratatui). Same error

**Reply by esahmed21** [reply]
Hi @DasLixou 👋

We’ve successfully addressed the issue, and the fix will be available shortly, Stay tuned. 

Thank you for your patience!

**Comment by SwiftLion23** [comment]
Thanks for fixing issue of  Mobile [app ](https://teatvhd.org/)crashes on loading
